src/resources/views/crud/inc/form_fields_script.blade.php

```diff
@@ -84,6 +83,10 @@ class CrudField {
             return this.change(closure);
         }
 
+        triggerChange() {
```

Comment:
I'm not 100% sure this is the best name for this method. I've also taken into consideration:
- `.change()`
- `.go()`
- `.init()`
- `.trigger()`
- `.start()`

But out of all of them... `triggerChange()` best explains what the method actually does...